onnxruntime/core/framework/cloud_invoker.cc

```diff
@@ -25,6 +26,8 @@ const char* kAzureVerbose = "azure.verbose";
 const char* kAzureEndpointType = "azure.endpoint_type";
```

Comment:
All of them should be "constexpr const char*".
Or you may use std::string_view . See ["Other Common Mistakes"](https://abseil.io/tips/140) section for more details.

Reply:
Done.